python/lsst/obs/lsstSim/processCalibLsstSim.py

```diff
@@ -120,17 +122,17 @@ def run(self, sensorRefList, calibType):
             if calibType is 'flat':
                 exp.setFilter(expfilter)
             if self.config.doWrite and calibType is not 'flat':
-                print "writing file %s"%dataId
-                butler.put(exp, calibType, dataId = amp.dataId)
+                print("writing file %s"%dataId)
+                butler.put(exp, calibType, dataId=amp.dataId)
             masterExpList.append(exp)
             dataIdList.append(amp.dataId)
         if self.config.doWrite and calibType is 'flat':
             self.normChipAmps(masterExpList)
             for exp, dataId in zip(masterExpList, dataIdList):
-                print "writing flat file %s"%dataId
+                print("writing flat file %s"%dataId)
```

Comment:
The % symbol should have spaces around it when used for formatting print statements. And more equals signs just below this too!